main.go

```diff
@@ -223,6 +242,14 @@ func main() {
 	setupLog.Info(fmt.Sprintf("Go OS/Arch: %s/%s", runtime.GOOS, runtime.GOARCH))
 	setupLog.Info(fmt.Sprintf("Running on Kubernetes %s", kubeVersion.PrettyVersion), "version", kubeVersion.Version)
 
+	ctx := context.Background()
```

Comment:
we shouldn't create a new context here, but we should use the same that's being created on line 253 : `if err := mgr.Start(ctrl.SetupSignalHandler()); err != nil {`

Reply:
Updated, thanks!